crates/astria-sequencer/src/fees/storage/keys.rs

```diff
@@ -18,34 +18,34 @@ pub(in crate::fees) const BRIDGE_UNLOCK: &str = "fees/bridge_unlock";
 pub(in crate::fees) const BRIDGE_SUDO_CHANGE: &str = "fees/bridge_sudo_change";
 pub(in crate::fees) const IBC_RELAY: &str = "fees/ibc_relay";
 pub(in crate::fees) const VALIDATOR_UPDATE: &str = "fees/validator_update";
-pub(in crate::fees) const FEE_ASSET_CHANGE: &str = "fees/fee_asset_change";
+pub(in crate::fees) const FEE_ASSET_CHANGE: &str = "fees/allowed_asset_change";
```

Comment:
This one should probably remain:
```suggestion
pub(in crate::fees) const FEE_ASSET_CHANGE: &str = "fees/fee_asset_change";
```